src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -262,9 +274,108 @@ const ProteinExternalReference = (props: ProteinExternalReferenceProps) => {
 
   return (
     <div className={styles.proteinExternalReference}>
-      <ExternalReference label={props.source} to={url} linkText={props.name} />
+      <ExternalReference
+        label={props.source}
+        to={url}
+        linkText={props.accessionId}
+      />
     </div>
   );
 };
 
+type ProteinExternalReferenceGroupProps = {
+  source: ExternalSource;
+  xrefs: Array<
+    Pick<ExternalReferenceType, 'accession_id' | 'name'> &
+      Pick2<ExternalReferenceType, 'source', 'id'>
+  >;
+};
+
+const ProteinExternalReferenceGroup = (
+  props: ProteinExternalReferenceGroupProps
+) => {
+  const { source, xrefs } = props;
+
+  const [isXrefGroupOpen, setXrefGroupOpen] = useState(false);
+
+  const toggleXrefGroup = () => {
+    setXrefGroupOpen(!isXrefGroupOpen);
+  };
+
+  const chevronClass = classNames(
+    styles.chevron,
+    isXrefGroupOpen? styles.chevronDown : styles.chevronUp
```

Comment:
A bit similar to the whitespace comment above. This time there is no space between `isXrefGroupOpen` and `?`. Isn't Prettier working in your VS Code?